data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question, looking at you instead, and asking, "You're with the Navy, aren't you? Finally figured out I was here?"`
+			branch navy
+				has "license: Navy"
+			choice
+				`	"No, I'm not with the Navy. You could've just asked without slamming me to the ground."`
+					goto notnavy
+				`	"Is that why you attacked me?"`
+					goto detained
+			label navy
+			choice
+				`	"I am with the Navy, but I didn't come here looking for you."`
+				`	"Is that why you attacked me?"`
+					goto detained
+			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip, and opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" The Alpha shouts, stopping his attempts to break free.`
+				goto detained
+			label notnavy
+			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
+			label detained
+			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
+			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells, kept shut by thick, tough walls of some transparent material, with a few holes for airflow. They lock him in one of the cells, making him the only prisoner at the moment. They then ask that you wait a few minutes for a medic to arrive, to take care of your bruises.`
+			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`
```

Comment:
If the agents took him away, why are you down next to him now? Is the medical centre in the same hallway as the cell block?

Reply:
Because I needed some way to get the talk started.
If having the medic's place be down by the cells would help, can do that.

Reply:
Yeah, that would make it make more sense.

Reply:
I've added mention of it at the end of the mission.